Libraries/LibIPC/TransportSocketWindows.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2024, Andrew Kaster <andrew@ladybird.org>
+ * Copyright (c) 2024, stasoid <stasoid@yahoo.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibIPC/TransportSocketWindows.h>
+// This comment stops clang-format from changing header order
+#include <AK/windows.h>
+#pragma comment(lib, "ws2_32.lib")
+
+namespace IPC {
+
+static int init_wsa()
```

Comment:
In my code I already have WSAStartup in main. Multiple startups are allowed, as long as a matching number of shutdowns gets called. Question is do we need to start it multiple times. And if we don't main is probably the better place to start it.